src/lib/prepareOas.ts

```diff
@@ -14,12 +14,12 @@ type FileSelection = {
 /**
  * Normalizes, validates, and (optionally) bundles an OpenAPI definition.
  *
- * @param {String} path path to spec file. if this is missing, the current directory is searched
- * for certain file names
- * @param command string to distinguish if it's being run in
- * an 'openapi' or 'validate' context
+ * @param path Path to a spec file. If this is missing, the current directory is searched for
+ *    certain file names.
+ * @param action The action context in which this is being run within (uploading a spec,
+ *    validation, or reducing one).
  */
-export default async function prepareOas(path: string, command: 'openapi' | 'validate') {
+export default async function prepareOas(path: string, command: 'openapi' | 'openapi:reduce' | 'validate') {
```

Comment:
ugh i'm SO excited that we're able to find yet another useful command that can piggyback off of the spec-finding logic 🥹